benchmarks/storage/benchmark_sample_latency_over_rpc.py

```diff
@@ -0,0 +1,183 @@
+"""
+Sample latency benchmarking (using RPC)
+======================================
+A rough benchmark of sample latency using different storage types over the network using `torch.rpc`.
+This code is based on examples/distributed/distributed_replay_buffer.py.
```

Comment:
Can you provide a simple example on how to run that script (since you need to run it twice with different ranks)?

Reply:
Hopefully this works